scripts/gitlab-mirror/mirror

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+TARGET=$1
+MIRROR=$2
+DELAY=${3:-60}
+SSH_KEY=${4:-/root/.ssh/gitlab-mirror}
```

Comment:
A neater way (IMHO) is to use  `~/.ssh/config`  to specify keys on a per host basis. Although slightly less flexible it does separate the transport layer from the script... just my preference ;)

Reply:
`~/.ssh/config` is necessary for this to work, and is currently meant to be included as part of the volume mount. I'll tidy this part of the code up so to make it clearer, and maybe avoid using a mount if I can.

They key was only there because it wasn't being added to the ssh-agent on startup for some reason. I'll do some googling as to the correct method of adding ssh keys to containers.